src/docs/JobPlanner/references/JobPlannerServiceReference.adoc

```diff
@@ -299,4 +299,145 @@ In such case, an *Association failed* notification will be triggered to users su
+
+To manage an association's authorizations in the Job-Planner Calendar Associations portal, first select the association, then click on the image:../../images/Open-authorisations-view.png[] above the associations view as shown in the figure below:
+
+image::../../images/Association-authorisations-button.png[align=center]
+
+A dedicated authorization view will open.
+The figure below shows an example:
+
+image::../../images/Association-authorisations-view.png[align=center]
+
+The authorization view consists in two parts:
+
+* The buttons on top allow to refresh the authorizations list, add a new authorization and delete an existing one.
+* The list below shows the authorization rules that are assigned to the selected association.
+
+Each authorization has the following properties:
+
+* Association ID
+* Grantee (required): the username or the user group name
+* Grantee Type: group or user
+* Access Type: `read`, `write`, `admin` or `noAccess`
+* Creator: The user who created the authorization
+* Priority: This attribute is only used for access associated with a group.
+For example, if the user belongs to 2 groups and each group has a different type of access, the system will choose the access with the highest priority for the user.
+For an authorization assigned to a user, the value is 0, since it will be ignored when calculating the resulting access type.
+
+To create a new authorization, the admin just needs to click on the image:../../images/Add-authorisation-button.png[] button.
+Then the view of creating an authorization (as shown in the figure below) will be popped up.
+The admin needs to fill in the authorization attributes (as presented above), then click the "Add authorization" button.
+The figure below shows an example of an authorization which gives the user named `Olivia` `write` access to association `1`.
+
+image::../../images/Create-association-authorisation.png[align=center]
+
+Once the admin has created an authorization, it will appear in the authorizations list in the authorization view, as shown in the figure below.
+
+image::../../images/List-of-authorisations.png[align=center]
+
+Once the first authorization has been created, the admin will see a "shield" icon next to the association to which the authorization was created.
+The figure below shows an example.
+
+image::../../images/Admin-association-shield.png[align=center]
+
+Moreover, when the user, who is benefiting from the authorization, log in to his/her account, he/she will be able to visualize an icon indicating the rights he/she possess over the association(s).
+These icons are the following:
+
+image::../../images/Association-authorrizations-icons.png[align=center]
+
+* The first icon indicates that the user has `read` rights over the association(s).
+* The second icon indicates that the user has `write` rights over the association(s).
+* The third icon indicates that the user has `admin` rights over the association(s).
```

Comment:
there is no icon for admin, which icon do you refer to ?

Reply:
if you mean the "with grants" icon, then you should rephrase:

* The third icon indicates that authorizations rules are defined on the association and the user has `admin` rights over the association.